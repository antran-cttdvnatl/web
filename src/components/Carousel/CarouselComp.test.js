import React from 'react';
import renderer from 'react-test-renderer';
import CarouselComp from './CarouselComp';


it('Carousel should render correctly', () => {
    const component = renderer.create(
        <CarouselComp />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})