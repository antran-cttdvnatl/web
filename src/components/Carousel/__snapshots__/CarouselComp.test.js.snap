// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel should render correctly 1`] = `
<div>
  <div
    className="carousel slide"
    onKeyDown={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <ol
      className="carousel-indicators"
    >
      <li
        className="active"
        onClick={[Function]}
      />
       
      <li
        className={null}
        onClick={[Function]}
      />
       
      <li
        className={null}
        onClick={[Function]}
      />
       
    </ol>
    <div
      className="carousel-inner"
    >
      <div
        className="carousel-item active carousel-item"
      >
        <img
          alt="first slide"
          className="carousel-image"
          src="./images/internal.jpg"
        />
        <div
          className="carousel-caption"
        >
          <h3>
            First Slide
          </h3>
          <p>
            easy on me
          </p>
        </div>
      </div>
      <div
        className="carousel-item carousel-item"
      >
        <img
          alt="Second slide"
          className="carousel-image"
          src="./images/external.jpg"
        />
        <div
          className="carousel-caption"
        >
          <h3>
            Second Slide
          </h3>
          <p>
            easy on me slide 2
          </p>
        </div>
      </div>
      <div
        className="carousel-item carousel-item"
      >
        <img
          alt="third slide"
          className="carousel-image"
          src="./images/external.jpg"
        />
        <div
          className="carousel-caption"
        >
          <h3>
            third Slide
          </h3>
          <p>
            easy on me slide 3
          </p>
        </div>
      </div>
    </div>
    <a
      className="carousel-control-prev"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
      <span
        className="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      className="carousel-control-next"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
      <span
        className="sr-only"
      >
        Next
      </span>
    </a>
  </div>
</div>
`;
